doc/managing-wallet.md

```diff
@@ -0,0 +1,83 @@
+# Managing the Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating the Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the `createwallet` RPC. More information about this command may be found by running `bitcoin-cli help createwallet`.
```

Comment:
I think it would be useful to provide instructions for creating a wallet via the GUI.

Reply:
Done.